templates/view_spec_helpers.rb

```diff
@@ -0,0 +1,21 @@
+module ViewSpecHelpers
+  def page
```

Comment:
Capybara has some Rails integration already. Maybe they'd be interested in a pull request which includes this method in view specs?